src/dc_receive_imf.rs

```diff
@@ -3533,4 +3533,70 @@ YEAAAAAA!.
         assert_eq!(chat.typ, Chattype::Single);
         assert_eq!(msg.get_text().unwrap(), "private reply");
     }
+
+    #[async_std::test]
+    async fn test_save_mime_headers_off() {
+        let alice = TestContext::new_alice().await;
+        let bob = TestContext::new_bob().await;
+        let chat_alice = alice.create_chat(&bob).await;
+        chat::send_text_msg(&alice, chat_alice.id, "hi!".to_string())
+            .await
+            .ok();
```

Comment:
This should be `unwrap`, not `ok`. I prefer returning `anyhow::Result<()>` from the test and using `?`, this makes it more clear because `unwrap` is related to `Option` and `?` is related to `Result` then.

Reply:
usually, i use `unwrap()` in test, not sure why i wrote `ok()`.

but the approach with `anyhow::Result<()>` seems to be even smarter. the test fail when return an error then, i assume.